scrapli/transport/plugins/system/transport.py

```diff
@@ -101,14 +101,31 @@ def _build_open_cmd(self) -> None:
         else:
             self.open_cmd.extend(["-o", "StrictHostKeyChecking=yes"])
             if self.plugin_transport_args.ssh_known_hosts_file:
-                self.open_cmd.extend(
-                    ["-o", f"UserKnownHostsFile={self.plugin_transport_args.ssh_known_hosts_file}"]
-                )
-
-        if self.plugin_transport_args.ssh_config_file:
-            self.open_cmd.extend(["-F", self.plugin_transport_args.ssh_config_file])
+                if (
```

Comment:
two things here:

1) if possible and we can flatten things a bit, lets do that :P (sorry for nit picks but flat is good :D)

2) should we have a second magic string for this since users could set known hosts -> True but not ssh config file? (I think? its been a while, maybe im wrong about that too hah)

Reply:
Flattened out, and good point that we should add a separate variable for the known_hosts_file ( I was being lazy :grin: ) added it!